keps/0036-conditional-operator-dependencies.md

```diff
@@ -0,0 +1,106 @@
+---
+kep-number: 36
+short-desc: Allowing existence of an operator dependency to be conditional on a parameter value
+title: Conditional Operator Dependencies
+authors:
+  - "@asekretenko"
+owners:
+  - "@asekretenko"
+editor: @asekretenko
+creation-date: 2021-04-30
+last-updated: 2021-04-30
```

Comment:
could we update this?

Reply:
Updated. Sorry, forgot about this timestamp.